stable/aws-node-termination-handler/Chart.yaml

```diff
@@ -1,5 +1,27 @@
 apiVersion: v1
-appVersion: "1.0"
-description: A Helm chart for Kubernetes
 name: aws-node-termination-handler
+description: A Helm chart for the AWS Node Termination Handler
 version: 0.1.0
+appVersion: 1.0.0
+home: https://github.com/aws/eks-charts
```

Comment:
I think the home is https://github.com/aws/aws-node-termination-handler no?

Reply:
I used the appmesh-jaeger Chart as a reference:

https://github.com/aws/eks-charts/blob/master/stable/appmesh-jaeger/Chart.yaml#L6